scrapers_next/wa/people.py

```diff
@@ -0,0 +1,167 @@
+import re
+from spatula import HtmlListPage, CSS, XPath, URL
+from ..common.people import ScrapePerson
+
+
+class PartyAugmentation(HtmlListPage):
+    """
+    WA Email addresses are listed on a separate page.
+    """
+
+    source = URL("https://app.leg.wa.gov/memberemail/Default.aspx")
+
+    def find_rows(self):
```

Comment:
might be better expressed as        

 return CSS("#membertable tbody tr").match(self.root, num_items=147)